wrapper.js

```diff
@@ -33,6 +33,14 @@ function setupMethods (soljson) {
           error: 'File import callback not supported'
         };
       };
+    } else if (typeof readCallback === 'object') {
+      if (readCallback.imports === undefined) {
```

Comment:
Is "imports" the name we want to go with?

Reply:
I think that's fine.
Not super important, but to avoid a little code repetition you could do like https://github.com/ethereum/solc-js/pull/382/files#diff-2f0f3cebee013143afb9e2f50c940f8eR30 which assigns `readCallback` `undefined` before checking the argument's type, and then creates a fallback callback if it's still undefined.

Reply:
How about the simplification I've pushed now?

Reply:
Nice!